modules/pyclone_vi/1.0/envs/gamblr.yaml

```diff
@@ -0,0 +1 @@
+/home/lhilton/repos/lcr-modules-starfish/envs/GAMBLR/gamblr.yaml
```

Comment:
This seems to point to home directory. Maybe this file needs to be updated (or removed)?

Reply:
The prefix is ignored when building envs.